perception/object_recognition/detection/cluster_data_association/include/cluster_data_association/node.hpp

```diff
@@ -12,44 +12,43 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 #pragma once
-#include "pcl/point_types.h"
+// #include "pcl/point_types.h"
 #include "pcl_conversions/pcl_conversions.h"
-#include "pcl_ros/point_cloud.h"
-#include "pcl_ros/transforms.h"
-#include "ros/ros.h"
+// #include "pcl_ros/point_cloud.h"
```

Comment:
Can these be deleted - or do you want to leave for when pcl_ros is ported?

Reply:
@esteve Just a small question - otherwise ready to be merged.

Reply:
They are not needed in fact, I've removed them in bd7fc1d